### PR TITLE
Add hammer_cli_foreman_webhooks

### PR DESCRIPTION
**Pull request by ofedoren**
Supported Versions:

 * Nightly
 * 2.5

I've tried to add 0.0.1 [1], but I've closed that in favor of this one. Needs to be CPed for 2.5.

[1] - https://github.com/theforeman/foreman-packaging/pull/6316

### CODE REVIEW

File: dependencies/buster/hammer_cli_foreman_webhooks/control [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Package: ruby-hammer-cli-foreman-webhooks
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ruby | ruby-interpreter,
+         ruby-hammer-cli-foreman (<< 4.0.0),
+         ruby-hammer-cli-foreman (>= 2.0.0),
+         ${misc:Depends},
+         ${shlibs:Depends}
+Description: Foreman Webhooks plugin for Hammer CLI
+ <insert long description, indented with spaces>
```
Comment (adamruzicka) [inline dependencies/buster/hammer_cli_foreman_webhooks/control:26]: This should probably go away

File: dependencies/focal/hammer_cli_foreman_webhooks/control [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Package: ruby-hammer-cli-foreman-webhooks
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ruby | ruby-interpreter,
+         ruby-hammer-cli-foreman (<< 4.0.0),
+         ruby-hammer-cli-foreman (>= 2.0.0),
+         ${misc:Depends},
+         ${shlibs:Depends}
+Description: Foreman Webhooks plugin for Hammer CLI
+ <insert long description, indented with spaces>
```
Comment (adamruzicka) [inline dependencies/focal/hammer_cli_foreman_webhooks/control:26]: This too

**Comment by ofedoren** [comment]
Thanks, @adamruzicka, updated.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Built in https://ci.theforeman.org/job/packaging_build_deb_dependency/522/

**Pull request merged**